package.json

```diff
@@ -279,6 +279,19 @@
             "3.0.0"
           ],
           "markdownDescription": "The Scala compiler version that is used as the default or fallback in case a file doesn't belong to any build target or the specified Scala version isn't supported by Metals.\n\nThis applies to standalone Scala files, worksheets, and Ammonite scripts.\n\n The `automatic` value means that the Scala version for these files will be inferred from the highest supported Scala version in your projects build definition"
+        },
+        "metals.eclipseFormatConfigPath": {
```

Comment:
Could we somehow squash those settings into one object maybe ? Or perhaps could the other two settings be included in the actual config?

It seems weird that there are settings that people need to set manually.